web/pdf_viewer.css

```diff
@@ -46,6 +46,46 @@
   border: none;
 }
 
+.pdfViewer.scrollHorizontal, .pdfViewer.scrollWrapped, .spread {
+  text-align: center;
+}
+
+.pdfViewer.scrollHorizontal, .pdfViewer.scrollWrapped {
+  margin: 4px;
+}
+
+.pdfViewer.scrollHorizontal, .spread {
+  white-space: nowrap;
+}
+
+.pdfViewer.removePageBorders {
+  margin: 0;
+}
+
+.spread .page,
+.pdfViewer.scrollHorizontal .page,
+.pdfViewer.scrollWrapped .page,
+.pdfViewer.scrollHorizontal .spread,
+.pdfViewer.scrollWrapped .spread {
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.spread .page {
+  margin: 1px -3.5px -8px;
```

Comment:
It seems that the intention of this CSS change is to avoid a big gap in pages within a spread.
The `1px` and `-8px` (respectively top and bottom) are identical to the default `.pdfViewer .page` values, so I guess that you intent to keep them the same.

Is there a reason to override `margin` instead of `margin-left` and `margin-right` individually?

Reply:
If you've read my ramble in the comment above this, you understand why I think the margin situation here is a little tricky. Given that, I thought it best to be explicit about what all the margins are in each case, rather than spreading it around more by putting left/right and top/bottom in different sets of selectors. There's no technical need to do this, so if you really prefer seeing it the other way I can make that modification.

Reply:
Sure, will do (especially given the change to only touch horizontal margins throughout).